### ISSUE TITLE
Deploy the simulation in a webpage

### ISSUE THREAD

**Issue by HadadIdan** [TechnionYP5779/SimuMole#92]
I will work on the code, while using Michael to test it (since my pc can't run the simulation)

**Comment by HadadIdan** [comment]
Update:
We have successfully found a way to show an interactive, 3d view of both the molecules, however there doesn't seem to be a solution for an interactive, 3d view on the MD itself ( = We can only see the first frame, but not the whole interaction between the molecules).

That being said, we can create a movie (.mp4 , .avi etc) of the MD. The downside is that the viewer can't rotate or move the camera since it's just an animation.

I propose the following solution:
Use the interactive 3d view of both molecules right after choosing their offsets, enabling the user to
see a preview of their location relative to one another, and then give him the possibility to go back and choose other offsets, or continue to render the animation.
After he sets the preferable offsets, show a number of animations (6 for example, as in 6 wigs of a cube) , each from a different camera angle, all running at the same time.
This way, although the user doesn't have the ability to rotate and move the camera, he sees few angles at the same time.

Another idea was to show only one angle, and have buttons for switching between them (and they
will "jump" to the exact point the previous animation [from the previous angle] was stopped).

To sum up, two options:
![image](https://user-images.githubusercontent.com/44426929/57471519-f785a280-7293-11e9-819a-19ed4bddf103.png)


Please let me know what you think about it, and which way of displaying the angles you think is best.
@MichaelShohat @EyalFaygen @IlanitSmul @Nitzan810 @OrYasso

**Comment by HadadIdan** [comment]
Closed as it was continued in #102
